test/conformance/device_code/CMakeLists.txt

```diff
@@ -113,7 +113,7 @@ macro(add_device_binary SOURCE_FILE)
             ${AMD_TARGET_BACKEND} ${AMD_OFFLOAD_ARCH} ${AMD_NOGPULIB}
             ${DPCXX_BUILD_FLAGS_LIST} ${SOURCE_FILE} -o ${EXE_PATH}
 
-            COMMAND ${CMAKE_COMMAND} -E env ${EXTRA_ENV} ${UR_DEVICE_CODE_EXTRACTOR} --stem="${TRIPLE}.bin" ${EXE_PATH}
+            COMMAND ${CMAKE_COMMAND} -E env ${EXTRA_ENV} ${UR_DEVICE_CODE_EXTRACTOR} --stem="${TRIPLE}.bin" ${EXE_PATH} >> /dev/null
```

Comment:
Not sure this is very portable, Windows doesn't have `/dev/null`. That's unlikely to get caught by CI since we don't have Windows HW runners, unfortunately.